connect/connect-c8/element-template-c8/src/main/java/io/miragon/miranum/connect/adapter/in/c8/elementtemplates/schema/Constraints.java

```diff
@@ -0,0 +1,232 @@
+
+    /**
+     * constraints minLength
+     * <p>
+     * The minimal length of a control field value
+     */
+    @JsonProperty("minLength")
+    @JsonPropertyDescription("The minimal length of a control field value")
+    private Double minLength;
+
+    /**
+     * constraints maxLength
+     * <p>
+     * The maximal length of a control field value
+     */
+    @JsonProperty("maxLength")
+    @JsonPropertyDescription("The maximal length of a control field value")
+    private Double maxLength;
+
+    /**
+     * constraints pattern
+     * <p>
+     * A regular expression pattern for a constraint
+     */
+    @JsonProperty("pattern")
+    @JsonPropertyDescription("A regular expression pattern for a constraint")
+    private String pattern;
+
+    /**
+     * constraints notEmpty
+     * <p>
+     * The control field must not be empty
+     */
+    @JsonProperty("notEmpty")
+    public Boolean getNotEmpty()
+    {
+        return notEmpty;
+    }
+
+    /**
+     * constraints notEmpty
+     * <p>
+     * The control field must not be empty
+     */
+    @JsonProperty("notEmpty")
+    public void setNotEmpty(Boolean notEmpty)
+    {
+        this.notEmpty = notEmpty;
+    }
+
+    public Constraints withNotEmpty(Boolean notEmpty)
+    {
+        this.notEmpty = notEmpty;
+        return this;
+    }
+
+    /**
+     * constraints minLength
+     * <p>
+     * The minimal length of a control field value
+     */
+    @JsonProperty("minLength")
+    public Double getMinLength()
+    {
+        return minLength;
+    }
+
+    /**
+     * constraints minLength
+     * <p>
+     * The minimal length of a control field value
+     */
+    @JsonProperty("minLength")
+    public void setMinLength(Double minLength)
+    {
+        this.minLength = minLength;
+    }
+
+    public Constraints withMinLength(Double minLength)
+    {
+        this.minLength = minLength;
+        return this;
+    }
+
+    /**
+     * constraints maxLength
+     * <p>
+     * The maximal length of a control field value
+     */
+    @JsonProperty("maxLength")
+    public Double getMaxLength()
+    {
+        return maxLength;
+    }
+
+    /**
+     * constraints maxLength
+     * <p>
+     * The maximal length of a control field value
+     */
+    @JsonProperty("maxLength")
+    public void setMaxLength(Double maxLength)
+    {
+        this.maxLength = maxLength;
+    }
+
+    public Constraints withMaxLength(Double maxLength)
+    {
+        this.maxLength = maxLength;
+        return this;
+    }
+
+    /**
+     * constraints pattern
+     * <p>
+     * A regular expression pattern for a constraint
+     */
+    @JsonProperty("pattern")
+    public String getPattern()
+    {
+        return pattern;
+    }
+
+    /**
+     * constraints pattern
+     * <p>
+     * A regular expression pattern for a constraint
+     */
+    @JsonProperty("pattern")
+    public void setPattern(String pattern)
+    {
+        this.pattern = pattern;
+    }
+
```

Comment:
builder pattern can also be added with lomok annotations

Reply:
I use the `@Accessor` instead since `@Builder` creates immutable fields.